packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -176,7 +176,7 @@ module.exports = ctx => ({
     ],
     primary_right_nav: [
       { text: 'Okta.com', target: '_blank', link: 'https://www.okta.com/' },
-      { text: 'Sign in to Okta', target: '_blank', link: '/login/' },
+      { text: 'Log in', target: '_blank', link: '/login/' },
```

Comment:
This should still remain _Sign in_. The page this links to uses Sign in.

Reply:
Per conversation with @chelseajan we will proceed with 'Log in' for now and change later if need be.